bitmap-steven-hawa-caleb/test/grayscale-test.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const fs = require('fs');
+const grayscaleColors = require(`${__dirname}/../lib/grayscale-colors.js`);
+const expect = require('chai').expect;
+
+describe('Grayscale Manipulation', function() {
+  describe('#grayscaleTransform', function() {
+    it('Should transform data into a grayscale.', function(done) {
+      grayscaleColors.grayscaleTransform(function(err, buffer) {
+        var checkOutliers = false;
+        for (var i = 54; i < 1078; i+=4) {
```

Comment:
@maschigokae @BatemanVO @abdih17 best to abstract your loop logic out of the test, we'll talk more about this next week